### PR TITLE
change entrypoint of nextcloud container to root

### PR DESCRIPTION
**Pull request by szaimen**
Close https://github.com/nextcloud/all-in-one/issues/1233
Close https://github.com/nextcloud/all-in-one/issues/1180

Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

File: Containers/nextcloud/notify.sh [hunk 1]
```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
 
+if [[ "$EUID" = 0 ]]; then
+    sudo -u www-data -s
```
Comment (szaimen) [inline Containers/nextcloud/notify.sh:4]: maybe needs an -E here additioinally

**Review by juliushaertl — APPROVED** [review]
Looks reasonable 👍

File: Containers/nextcloud/start.sh [hunk 2]
```diff
@@ -28,8 +28,16 @@ if [ -n "$TRUSTED_CACERTS_DIR" ]; then
     update-ca-certificates
 fi
 
+# Check datadir permissions
+sudo -u www-data touch "$NEXTCLOUD_DATA_DIR/this-is-a-test-file" &>/dev/null
+if ! [ -f "$NEXTCLOUD_DATA_DIR/this-is-a-test-file" ]; then
+    chown -R www-data:root "$NEXTCLOUD_DATA_DIR"
+    chmod 750 -R "$NEXTCLOUD_DATA_DIR"
+fi
+rm -f "$NEXTCLOUD_DATA_DIR/this-is-a-test-file"
```
Comment (szaimen) [inline Containers/nextcloud/start.sh:37]: Should also use the user

File: Containers/nextcloud/notify.sh (hunk 1 above)
Comment (szaimen) [reply Containers/nextcloud/notify.sh]: done

File: Containers/nextcloud/start.sh (hunk 2 above)
Comment (szaimen) [reply Containers/nextcloud/start.sh]: done

**Pull request merged**